charts/memgraph/templates/statefulset.yaml

```diff
@@ -69,7 +69,14 @@ spec:
               add: ["CHOWN"]
             runAsUser: 0
             runAsNonRoot: false
-
+        {{- if .Values.sysctlInitContainer.enabled }}
+        - name: init-sysctl
+          image: busybox
```

Comment:
Busybox is for shell executables right?

Reply:
Yes, I have tested this locally. We could set up a test for this, but I am not sure that is necessary.